dev/dags/dbt/jaffle_shop_python/.user.yml

```diff
@@ -0,0 +1 @@
+id: 215ac1e7-601b-45dd-9867-587a3e282bce
```

Comment:
@abhi12mohan please, could you remove this file, it shouldn't be committed - I'll do a separate PR to add it to `.gitignore`